integrations/SD_Lon/test_sd_connectivity.py

```diff
@@ -0,0 +1,142 @@
+import json
+import pathlib
+from uuid import UUID
+
+
+class TestSdConnectivity(object):
+    def __init__(self):
+        cfg_file = pathlib.Path.cwd() / 'settings' / 'settings.json'
+        if not cfg_file.is_file():
+            raise Exception('No setting file')
+        try:
+            self.settings = json.loads(cfg_file.read_text())
+        except json.decoder.JSONDecodeError as e:
+            print('Syntax error in settings file: {}'.format(e))
+            exit(1)
+
+    def _check_existens_of_keys(self):
```

Comment:
Det glæder jeg mig til vi gentager over hele linijen - så er det bare at slå alle behøvede jobs ind og læse checket.
Måske skal vi lave et 'check-job - som kører alle de (og kun de) behøvede checks. checks for alle de enablede jobs på en gang - uden at fejle imellem.

Dette job skal så også have et, der tester ALLE MO's og os2syncs interfaces.